types/simperium/index.d.ts

```diff
@@ -0,0 +1,228 @@
+// Type definitions for simperium 1.1
+// Project: https://github.com/Simperium/node-simperium#readme
+// Definitions by: Dennis Snell <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/simperium/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me